app/src/main/java/com/codesoom/assignment/application/product/ProductService.java

```diff
@@ -0,0 +1,15 @@
+package com.codesoom.assignment.application.product;
+
+import com.codesoom.assignment.domain.product.Product;
+import com.codesoom.assignment.dto.product.ProductData;
+
+import java.util.List;
+
+public interface ProductService {
+    List<Product>  getProducts();
```

Comment:
커밋하기 전에 포매터를 실행하는 습관을 갖거나 pre-commit 훅에 포매터를 추가해 보는 것도 좋겠죠.
```suggestion
    List<Product> getProducts();
```